massa-ledger-exports/src/mapping_grpc.rs

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2023 MASSA LABS <info@massa.net>
+
+use crate::{LedgerEntry, LedgerEntryUpdate, SetOrDelete, SetOrKeep};
+use massa_proto::massa::api::v1 as grpc;
+
+impl From<LedgerEntry> for grpc::LedgerEntry {
+    fn from(value: LedgerEntry) -> Self {
+        grpc::LedgerEntry {
+            balance: value.balance.to_raw(),
+            bytecode: value.bytecode.0,
+            entries: value
```

Comment:
The datastore doesn't hold a candidate and final value but a key and value. The ledger entries don't have the concept of candidate or final because they are just ledger entries that could be used in both context.

Reply:
done